src/Makefile

```diff
@@ -12,7 +12,7 @@ LINKFLAGS = -I $(CAMLP5D)
 include $(ROOT)/tools/Makefile.common
 
 compilation.ml: always
-	echo "Util.compilation_time_hook.val := \" ($$(date "+%Y-%m-%d %T %Z"))\";" > $@
+	echo "Util.compilation_time_hook.val := \" (compiled on $$(date "+%Y-%m-%d %T %Z"))\";" > $@
```

Comment:
I think you should not put that here because you lost the possibility for translation